test_requirements.txt

```diff
@@ -1,5 +1,4 @@
 freezegun==0.2.8
 mock==2.0.0
-mysqlclient==1.3.6
-psycopg2==2.6
+psycopg2==2.9.3
```

Comment:
`mysqlclient` gets removed here - but the travis file mentions set up with sql settings - am I missing something here?

Reply:
Travis file is no used unfortunately